.github/workflows/tests.yml

```diff
@@ -126,6 +126,40 @@ jobs:
           s3cmd --host localhost --host-bucket="localhost/%(bucket)" --access_key=fooAccessKey --secret_key=fooSecretKey --no-ssl put -P ~/test.txt s3://testbucket
           curl -s http://localhost/testbucket/test.txt | grep -F hello-radosgw
 
+      - name: Test Cluster Config
+        run: |
+          set -eux
+          cip=$(ip -4 -j route | jq -r '.[] | select(.dst | contains("default")) | .prefsrc' | tr -d '[:space:]')
+          
+          # pre config set timestamp for service age
+          ts=$(sudo systemctl show snap.microceph.osd.service -f json-pretty | grep -m1 ActiveEnterTimestampMonotonic | awk '{n=split($1,A,"="); print A[n]}')
+
+          # set config
+          sudo microceph cluster config set cluster_network $cip/8 --wait
+
+          # post config set timestamp for service age
+          ts2=$(sudo systemctl show snap.microceph.osd.service -f json-pretty | grep -m1 ActiveEnterTimestampMonotonic | awk '{n=split($1,A,"="); print A[n]}')
+
+          # Check config output
+          output=$(sudo microceph cluster config get cluster_network | grep -cim1 'cluster_network')
+          if [[ $output -ge 1 ]] ; then
+              echo "Config is present"
+          else
+              exit 1 # Config check failed.
```

Comment:
Think adding some explicit log output why the check failed would be good, maybe:

echo "config check failed: $output"

Reply:
ack